examples/v2/chat2.nim

```diff
@@ -31,7 +31,7 @@ const Help = """
 """
 
 const DefaultTopic = "waku"
-const DefaultContentTopic = "dingpu"
+const DefaultContentTopic = ContentTopic(1)
```

Comment:
Can we make the constructor us "dingpu" in byte format? Instead of using 1 everywhere.

Reply:
@oskarth yeah, was going to do that. Was just testing to see if everything works.